packages/helpers/src/dom/isElementContainsFocus.js

```diff
@@ -0,0 +1,4 @@
+export default function isElementContainsFocus(element) {
```

Comment:
`isElementFocussed` would read better.  But it's an internal name, not affecting the API, so let's update in the next PR.